src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -41,6 +41,23 @@
             }
         },
 
+        {
+            "Tests": [
+                {
+                    "Type": "FileExists",
+                    "Condition": {
+                        "FileName": "Dockerfile"
+                    }
+                }
+            ],
+            "Effect": {
+                "Fail": {
+                    "PriorityAdjustment": 100,
```

Comment:
any reason why you are modifying the weights?

Reply:
Because I switch the logic to raise the priority on Beanstalk recipe if there is no Dockerfile. That way the logic is just in the Beanstalk recipe instead of in all container recipes.